content/en/docs/languages/go/instrumentation.md

```diff
@@ -534,6 +534,35 @@ func removeItem() {
 }
 ```
 
+### Using Gauges
```

Comment:
```suggestion
### Using gauges
```

We use sentence case in titles.

Reply:
Thank you for the feedback. I noticed that other titles like 'Using Histograms' also start with capital letters.  Could you give me more detail on which part  i should fix?